lib/package.rb

```diff
@@ -227,7 +227,15 @@ def self.get_url(architecture)
   end
 
   def self.get_binary_url(architecture)
-    return @binary_url.key?(architecture) ? @binary_url[architecture] : nil
+    if @binary_sha256.key?(architecture)
+      if architecture == x86_64 || i686 || armv7l
+        return "https://gitlab.com/api/v4/projects/26210301/packages/generic/#{name}/#{version}_#{architecture}/#{name}-#{version}-chromeos-#{architecture}.#{binary_compression}"
+      elsif architecture == aarch64
+        return "https://gitlab.com/api/v4/projects/26210301/packages/generic/#{name}/#{version}_armv71/#{name}-#{version}-chromeos-armv71.#{binary_compression}"
```

Comment:
Where is `#{binary_compression}` set?

Reply:
In packages-- I just missed the commit last rebase.